updates/35/updates.sql

```diff
@@ -218,3 +218,85 @@ INSERT INTO assets(identifier, name, type) VALUES('CLOUD', 'Cloud', 'Y'); INSERT
+*
+INSERT INTO assets(identifier, name, type) VALUES('FRED', 'First Convicted RACCON', 'Y'); INSERT INTO common_asset_details(identifier, symbol, coingecko, cryptocompare, forked, started, swapped_for) VALUES('FRED', 'FRED', 'first-convicted-raccon', 'FRED', NULL, 1735713875, NULL);
+*
+INSERT INTO assets(identifier, name, type) VALUES('FTON', 'Fanton Token', 'W'); INSERT INTO common_asset_details(identifier, symbol, coingecko, cryptocompare, forked, started, swapped_for) VALUES('FTON', 'FTON', 'fanton-token', '', NULL, 1727851920, NULL);
+*
+INSERT INTO assets(identifier, name, type) VALUES('eip155:1/erc20:0x423071774C43c0Aaf4210b439E7CDA8c797e2F26', 'GALAXIS Token', 'C'); INSERT INTO evm_tokens(identifier, token_kind, chain, address, decimals, protocol) VALUES('eip155:1/erc20:0x423071774C43c0Aaf4210b439E7CDA8c797e2F26', 'A', 1, '0x423071774C43c0Aaf4210b439E7CDA8c797e2F26', 18, NULL); INSERT INTO common_asset_details(identifier, symbol, coingecko, cryptocompare, forked, started, swapped_for) VALUES('eip155:1/erc20:0x423071774C43c0Aaf4210b439E7CDA8c797e2F26', 'GALAXIS', 'galaxis-token', 'GALAXIS', NULL, 1714385903, NULL);
+*
+INSERT INTO assets(identifier, name, type) VALUES('eip155:8453/erc20:0x2D189eABb667aA1ecFC01963a6a3A5d83960f558', 'GALAXIS Token', 'C'); INSERT INTO evm_tokens(identifier, token_kind, chain, address, decimals, protocol) VALUES('eip155:8453/erc20:0x2D189eABb667aA1ecFC01963a6a3A5d83960f558', 'A', 8453, '0x2D189eABb667aA1ecFC01963a6a3A5d83960f558', 18, NULL); INSERT INTO common_asset_details(identifier, symbol, coingecko, cryptocompare, forked, started, swapped_for) VALUES('eip155:8453/erc20:0x2D189eABb667aA1ecFC01963a6a3A5d83960f558', 'GALAXIS', 'galaxis-token', 'GALAXIS', NULL, 1727265999, NULL);
+*
+INSERT INTO assets(identifier, name, type) VALUES('eip155:42161/erc20:0xA5312C3E42A82D459162B2a3bD7FfC4F9099b911', 'GALAXIS Token', 'C'); INSERT INTO evm_tokens(identifier, token_kind, chain, address, decimals, protocol) VALUES('eip155:42161/erc20:0xA5312C3E42A82D459162B2a3bD7FfC4F9099b911', 'A', 42161, '0xA5312C3E42A82D459162B2a3bD7FfC4F9099b911', 18, NULL); INSERT INTO common_asset_details(identifier, symbol, coingecko, cryptocompare, forked, started, swapped_for) VALUES('eip155:42161/erc20:0xA5312C3E42A82D459162B2a3bD7FfC4F9099b911', 'GALAXIS', 'galaxis-token', 'GALAXIS', NULL, 1714695650, NULL);
+*
+INSERT INTO assets(identifier, name, type) VALUES('eip155:137/erc20:0x3C69d114664d48357d820Dbdd121a8071eAc99bf', 'GALAXIS Token(PoS)', 'C'); INSERT INTO evm_tokens(identifier, token_kind, chain, address, decimals, protocol) VALUES('eip155:137/erc20:0x3C69d114664d48357d820Dbdd121a8071eAc99bf', 'A', 137, '0x3C69d114664d48357d820Dbdd121a8071eAc99bf', 18, NULL); INSERT INTO common_asset_details(identifier, symbol, coingecko, cryptocompare, forked, started, swapped_for) VALUES('eip155:137/erc20:0x3C69d114664d48357d820Dbdd121a8071eAc99bf', 'GALAXIS', 'galaxis-token', 'GALAXIS', NULL, 1714637112, NULL);
```

Comment:
```suggestion
INSERT INTO assets(identifier, name, type) VALUES('eip155:137/erc20:0x3C69d114664d48357d820Dbdd121a8071eAc99bf', 'GALAXIS Token (PoS)', 'C'); INSERT INTO evm_tokens(identifier, token_kind, chain, address, decimals, protocol) VALUES('eip155:137/erc20:0x3C69d114664d48357d820Dbdd121a8071eAc99bf', 'A', 137, '0x3C69d114664d48357d820Dbdd121a8071eAc99bf', 18, NULL); INSERT INTO common_asset_details(identifier, symbol, coingecko, cryptocompare, forked, started, swapped_for) VALUES('eip155:137/erc20:0x3C69d114664d48357d820Dbdd121a8071eAc99bf', 'GALAXIS', 'galaxis-token', 'GALAXIS', NULL, 1714637112, NULL);
```
There should probably be a space between `Token` and `(PoS)` here. Also, if we put PoS after some of the polygon pos tokens shouldn't they all have it? Its missing on `FARCANA` and `Matr1x Fire Token`.

Reply:
We always take the name from the name attribute on the token. PoS appears when the token is bridged from L1 and not native to the chain, that is why some have it and others don't. In arbitrum for example the same happens but they don't add anything new to the token symbol. There are native tokens and bridged tokens